components/dotcms/layout/Header.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Nav from './layout/Nav/Nav';
+import Nav from './Nav/Nav';
 
 function Header() {
     return <Nav />;
```

Comment:
This doesn't make sense, we should separate nav from header and also fix the `<a><a></a></a>` error.